core/src/main/java/io/questdb/TelemetrySystemEvent.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb;
+
+public final class TelemetrySystemEvent {
+    public static final short APPLIED_TXN = 103;
+    public static final short APPLY_STRUCTURE_CHANGE = 104;
+    public static final short APPLY_TXN_DATA = 105;
+    public static final short APPLY_TXN_SQL = 106;
```

Comment:
```suggestion
    public static final short WAL_APPLY_TXN_SQL = 106;
```

Reply:
`WAL_APPLY` was not in the name because there is an origin which tells that it is the apply job doing it.
with the removal of the `origin` field it makes sense to add it as a prefix to the events.